src/pages/SSL/locales/en-US.ts

```diff
@@ -15,39 +15,41 @@
  * limitations under the License.
  */
 export default {
-  'ssl.form.expiration.time':'Expiration Time',
-  'ssl.form.public.key':'Public Key',
+  'ssl.form.expiration.time': 'Expiration Time',
+  'ssl.form.cert': 'Cert',
   'ssl.form.check.cert.value': 'Please check Cert value',
   'ssl.form.cert.value.length': 'The Cert value requires at least 128 characters',
-  'ssl.form.private.key':'Private Key',
+  'ssl.form.private.key': 'Private Key',
   'ssl.form.check.key.value': 'Please check Key value',
   'ssl.form.key.value.length': 'The Key value requires at least 128 characters',
 
-  'ssl.upload.public.key':'Upload Public Key',
-  'ssl.upload.private.key':'Upload Private Key',
+  'ssl.upload.cert': 'Upload Cert',
+  'ssl.upload.private.key': 'Upload Private Key',
 
-  'ssl.step.way':'Way',
-  'ssl.step.rules':'The new certificate should contain the same about SNI as the current certificate',
-  'ssl.step.select.create.ways':'Please select create ways',
-  'ssl.step.input':'Input',
-  'ssl.step.upload':'Upload',
+  'ssl.step.way': 'Way',
+  'ssl.step.rules':
+    'The new certificate should contain the same about SNI as the current certificate',
+  'ssl.step.select.create.ways': 'Please select create ways',
```

Comment:
methods

Reply:
This section is a product of code formatting